lib/navigation/accelerometer_trajectory.hpp

```diff
@@ -0,0 +1,37 @@
+#pragma once
+
+#include <cstdint>
+#include <optional>
+
+#include "core/time.hpp"
+#include "core/timer.hpp"
+#include "core/types.hpp"
+
+namespace hyped::navigation {
+
+class AccelerometerTrajectoryEstimator {
+ public:
+  AccelerometerTrajectoryEstimator(const core::ITimeSource &time,
+                                   const core::TimePoint initial_time);
```

Comment:
are we not going to be logging anything here? `update` could surely have a kDebug log or two

Reply:
What would I need to log? This is just going to be done automatically on receiving new accelerometer data and all the maths is numerically stable so shouldn't really have issues with it. I can put a couple debug logs in if it's better but I didn't feel it was necessary?